app/services/search/search_service.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Search
+  class SearchService
+    def self.new(search_engine:, keyword:, language: 'en')
+      case search_engine
+      when 'google'
```

Comment:
We use `symbol` over `string` for these cases
```suggestion
      when :google
```

Reply:
@hoangmirs 👍 Thank you. Good to know.